src/content/en/updates/2020/10/http-cache-partitioning.md

```diff
@@ -0,0 +1,261 @@
+  <figcaption>
+    <b>Cache Key</b>: { <code>https://x.example/doge.png</code> }
+  </figcaption>
+</figure>
+
+A user visits a page (`https://a.example`) that requests an image
+(`https://x.example/doge.png`). The image is requested from the network and
+cached using `https://x.example/doge.png` as the key.
+
+<div class="clearfix"></div>
+
+<figure class="attempt-left">
+  <img src="/web/updates/images/2020/10/http-cache-partitioning-2.png">
+  <figcaption>
+    <b>Cache Key</b>: { <code>https://x.example/doge.png</code> }
+  </figcaption>
+</figure>
+
+The same user visits another page (`https://b.example`), which requests the same
+image (`https://x.example/doge.png`).  
+The browser checks its HTTP Cache to see
+if it already has this resource cached, using the image URL as the key. The
+browser finds a match in its Cache, so it uses the cached version of the
+resource.
+
+<div class="clearfix"></div>
+
+<figure class="attempt-left">
+  <img src="/web/updates/images/2020/10/http-cache-partitioning-3.png">
+  <figcaption>
+    <b>Cache Key</b>: { <code>https://x.example/doge.png</code> }
+  </figcaption>
+</figure>
+
+It doesn't matter if the image is loaded from within an iframe. If the user
+visits another website (`https://c.example`) with an iframe
+(`https://d.example`) and the iframe requests the same image
+(`https://x.example/doge.png`), the browser can still load the image from its
+cache because the cache key is the same across all of the pages.
+
+<div class="clearfix"></div>
+
+This mechanism has been working well from a performance perspective for a long
+time. However, the time a website takes to respond to HTTP requests can reveal
+that the browser has accessed the same resource in the past, which opens the
+browser to security and privacy attacks, like the following:
+
+- **Detection if a user has visited a specific site**: An adversary can detect a
+  user's browsing history by checking if the cache has a resource which might be
+  specific to a particular site or cohort of sites.
+- **[Cross-site search
+  attack](https://portswigger.net/daily-swig/new-xs-leak-techniques-reveal-fresh-ways-to-expose-user-information)**:
+  An adversary can detect if an arbitrary string is in the user's search results
+  by checking whether a 'no search results' image used by a particular website
+  is in the browser's cache or not.
+- **Cross-site tracking**: The cache can be used to store cookie-like
+  identifiers as a cross-site tracking mechanism.
+
+To mitigate these risks, Chrome will partition its HTTP cache starting in Chrome 86.
+
+## How will cache partitioning affect Chrome's HTTP Cache?
+
+With cache partitioning, cached resources will be keyed using a new "Network
+Isolation Key" in addition to the resource URL. The Network Isolation Key is
+composed of the top-level site and the current-frame site.
+
+Note: The "site" is recognized using "scheme://eTLD+1" so if requests are from
```

Comment:
I found information on eTLD. What does the "+1" mean? Is it something that needs an explanation.

Reply:
This is explained in the article linked at the last sentence.

Reply:
In which link? I can't find it.

Reply:
This is within a "Note" section and at the bottom of the box it's linked to https://web.dev/same-site-same-origin/